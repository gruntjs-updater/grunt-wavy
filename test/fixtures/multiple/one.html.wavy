<div class="multiple">
    @include test(One Test)
</div>