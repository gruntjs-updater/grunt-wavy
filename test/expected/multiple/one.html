<div class="multiple">
    
    <div>
    One Test
    </div>

</div>